packages/tempus/src/Tempus_StepperForwardEuler_decl.hpp

```diff
@@ -54,54 +68,30 @@ class StepperForwardEuler : virtual public Tempus::Stepper<Scalar>
       const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& appModel);
     virtual void setNonConstModel(
       const Teuchos::RCP<Thyra::ModelEvaluator<Scalar> >& appModel);
-    virtual Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >
-      getModel(){return appModel_;}
-
-    virtual void setSolver(std::string solverName);
-    virtual void setSolver(
-      Teuchos::RCP<Teuchos::ParameterList> solverPL=Teuchos::null);
-    virtual void setSolver(
-        Teuchos::RCP<Thyra::NonlinearSolverBase<Scalar> > solver);
-    virtual Teuchos::RCP<Thyra::NonlinearSolverBase<Scalar> > getSolver() const
-      { return Teuchos::null; }
+    //virtual Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >
```

Comment:
Is this needed anymore?

Reply:
Yeah, I went through and removed many functions that were covered by the StepperExplicit base class.  Good catch!